File: 2003/IntroToVBNETPart2/Tutorial/Fahrenheit3.vb
Imports System

Public Class Fahrenheit3
	Const LOWER As Integer = 0
	Const UPPER As Integer = 300
	Const LOOPSTEP As Integer = 20

	Shared Sub Main ()
		Dim fahr As Integer
		Dim celsius As Single

		For fahr = LOWER To UPPER Step LOOPSTEP
			celsius = CSng ( ( 5.0 / 9.0 ) * ( fahr - 32 ) )
			Console.WriteLine ( "{0,4:F0} {1,6:F1}", fahr, celsius )
		Next fahr
	End Sub
End Class

File: 2003/IntroToVBNETPart2/Tutorial/Fahrenheit3.cs
using System ;

public class Fahrenheit3
{
	const int LOWER = 0 ;
	const int UPPER = 300 ;
	const int STEP = 20 ;

	public static void Main ()
	{
		int fahr ;
		float celsius ;
		for ( fahr = LOWER ; fahr < UPPER ; fahr = fahr + STEP )
		{
			celsius = (float) ( 5.0 / 9.0 ) * ( fahr - 32 )  ;
			Console.WriteLine ( "{0,4:F0} {1,6:F1}", fahr, celsius ) ;
		}
	}
}